README.md

```diff
@@ -1,19 +1,20 @@
 <img alt="Vector logo" width="433" src="https://raw.githubusercontent.com/scikit-hep/vector/main/docs/_images/vector-logo.png"/>
 
-# Vector
+# Vector: arrays of 2D, 3D, and Lorentz vectors
+
+[![Scikit-HEP][sk-badge]][sk-link]
```

Comment:
The Scikit-HEP badge is always last, I believe. At least it's always in the last group. Actually, if the order of these two grapes was swapped, I think it would be pretty close. The idea was to have the standard "status" badges in the first row (like tests and such), and informational badges (like versions and DOI's) in the second row.

Reply:
Not sure I like, but it does not matter anyway. I prefer the way it is in Particle and DecayLanguage, which is what I followed.

Approved from you, then?

Reply:
Yes. I like the "status" badges on the top, since that's what the badges originally were for, and that's what users are used to seeing them used as. We've sort of "hijacked" them over time to show static links, but they were originally live statuses for CI. The lower/second line can be the "informational" one - expecially the scikit-hep badge, which is just a completely static link. Placing it completely last does give it a special spot, though.

Eventually we should add some recommendations and maybe even checking for the badges and their order...